tests/Neo.VM.Tests/Tests/OpCodes/BitwiseLogic/EQUAL.json

```diff
@@ -462,7 +462,7 @@
                 "evaluationStack": [
                   {
                     "type": "Boolean",
-                    "value": false
+                    "value": true
```

Comment:
Things like this are exactly that to me --- broken EQUAL. Because there was some well-defined and tested behavior for it that is now changed without any real reason.